### PR TITLE
add additional seed node

### PR DESCRIPTION
**Pull request by BigslimVdub**
Add additional seed node and update Copywrite year to 2022

### CODE REVIEW

File: src/cryptonote_config.h [hunk 1]
```diff
@@ -292,6 +292,7 @@ namespace arqma_nodes
     "207.244.249.105",
     "139.99.106.122",
     "164.68.123.118"
```
Comment (malbit) [inline src/cryptonote_config.h]: coma required at next to last line :)

File: src/cryptonote_config.h [hunk 2]
```diff
@@ -310,6 +311,7 @@ namespace arqma_nodes
     "it-support.mal-bit.com",
     "164.68.123.118",
     "144.217.242.16"
```
Comment (malbit) [inline src/cryptonote_config.h]: Same as above

**Review by malbit — CHANGES_REQUESTED** [review]
Did you forget about comas ??

**Comment by BigslimVdub** [comment]
Updated

**Review by malbit — APPROVED** [review]

**Pull request merged**